common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -458,6 +458,31 @@ def test_showanswer_finished(self):
                                             graceperiod=self.two_day_delta_str)
         self.assertTrue(still_in_grace.answer_available())
 
+    def test_showanswer_answered(self):
+        """
+        Tests that with showanswer="answered" should show answer after the problem is correctly answered.
```

Comment:
`showanswer="answered"` is there any alternative of this? As I don't know if  `showanswer` is property or some condition I will need to dig a little to find out what this test is for.
`showanswer` is also mentioned in the name of test.

Reply:
I was just matching the convention used in the previous tests for `showanswer` property. 
https://github.com/edx/edx-platform/pull/16319/files/fe5a8de55cf6f75f47acda5e4c314ba7d258caac#diff-e20c80a33d91d671320a98c86a7eac48R419